apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/notice.scss

```diff
@@ -17,6 +17,30 @@
 	}
 }
 
+.components-notice:has(.wpcom-global-styles-action-is-upgrade):not(:has(.wpcom-global-styles-action-is-support)) {
+	.components-notice__content {
+		display: flex;
+		flex-wrap: wrap;
+		margin-right: 0;
+		line-height: 36px;
```

Comment:
This is making the text to have too much space when wrapping to multiple lines:

Before | After
--- | ---
<img width="534" alt="Screenshot 2024-01-23 at 15 19 26" src="https://github.com/Automattic/wp-calypso/assets/1233880/5bf9f1a4-5786-4fc4-ab64-dc2779837a0b"> | <img width="526" alt="Screenshot 2024-01-23 at 15 19 30" src="https://github.com/Automattic/wp-calypso/assets/1233880/2399e353-cb71-4f6b-a8fb-30013c8891fb">

Reply:
This is done, by using `align-items: center` instead. Thanks!